### PR TITLE
task-aWiRf2kJ

### PR DESCRIPTION
**Pull request by BYavorskyj**
Added a text "Menu" before links
Style Menu maked text 16px and links 14px

https://trello.com/c/aWiRf2kJ/12-update-menu-component
@AntonShalenui @vkryviak-ua

### CODE REVIEW

File: src/client/components/Menu.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import "./Menu.css";
```
Comment (vkryviak-ua) [inline src/client/components/Menu.js]: Rename to menu.css

File: src/client/components/Menu.js [hunk 2]
```diff
@@ -9,11 +10,13 @@ export default class Menu extends Component {
       {name: 'Fourth link', url: 'https://www.linkedin.com/feed/'},
       {name: 'Fifth link', url: 'https://list.in.ua/%D0%9B%D1%8C%D0%B2%D1%96%D0%B2'},
     ],
+    menu:"Menu",
```
Comment (vkryviak-ua) [inline src/client/components/Menu.js]: Please use just simple text in render

Comment (BYavorskyj) [reply src/client/components/Menu.js]: @vkryviak-ua ,
explain it. 
Which is simple text?

File: src/client/components/Menu.js (hunk 1 above)
Comment (BYavorskyj) [reply src/client/components/Menu.js]: done

File: src/client/components/Menu.js (hunk 2 above)
Comment (vkryviak-ua) [reply src/client/components/Menu.js]: <p>Menu</p>

Comment (vkryviak-ua) [reply src/client/components/Menu.js]: You can merge PR after fix this

Comment (BYavorskyj) [reply src/client/components/Menu.js]: changed by another. look at https://github.com/vkryviak-ua/first-project/pull/13/files

**Comment by vkryviak-ua** [comment]
@BYavorskyj Here is no text and taskId is incorrect

File: src/client/components/Menu.js (hunk 2 above)
Comment (BYavorskyj) [reply src/client/components/Menu.js]: added

**Pull request merged**